common-utils/common-UI/common-UI.js

```diff
@@ -16,22 +16,23 @@ export class CommonUI {
 
   /**
    * Sets the page date range
+   * Sets start date first then end date
    * @param {String} start Start datetime to set
    * @param {String} end  End datetime to set
    */
-  setDateRange(start, end) {
+  setDateRange(end, start) {
```

Comment:
any reason we break the api contract?

Reply:
I think it is a tradeoff between keeping the general practice (as @seraphjiang mentioned above) and minimizing the amount of changes across multiple repos.

The current [consumers](https://github.com/search?q=org%3Aopensearch-project+setDateRange&type=code) of this repo are using the API in the end date first way. Thus this PR won't need them to make corresponding changes.

Reply:
not sure why need to reverse the order in the requirement, 

just my two cents if the assumption is that end  always greater start,  then we can implement the validation inside the setDateRange method:

if (end < start) {
    start, end = end, start
}